src/cgame/cgame.h

```diff
@@ -595,7 +595,7 @@ typedef struct cg_import_s {
 	 * @return The linked entity.
 	 * @remarks The parent entity must have been previously returned by `AddEntity`.
 	 */
-	const r_entity_t *(*AddLinkedEntity)(const r_entity_t *parent, const r_model_t *model, const char *tag_name);
+	r_entity_t *(*AddLinkedEntity)(const r_entity_t *parent, const r_model_t *model, const char *tag_name);
```

Comment:
So the cgame can mess with an entity that's already been copied into the renderer? Is this to allow tag alignment in cgame?